ERCS/erc-7628.md

```diff
@@ -0,0 +1,111 @@
+---
+eip: 7628
+title: ERC-721 Balance Extension
+description: Extend ERC-721 to include balance functionalities similar to ERC-20.
```

Comment:
Your description doesn't really include _that_ much more information than your title. Try to use your description to introduce more detail, and to differentiate it from other (possibly hypothetical) similar proposals.

ERC-721 already has a `balanceOf` function, so you really should explain what "balance functionalities" means.

Reply:
I have renamed the proposal to 'ERC-721 Ownership Shares Extension', which better aligns with the motivation.